blender/LilySurfaceScraper/Scrapers/HdriHavenScraper.py

```diff
@@ -81,7 +81,7 @@ def fetchVariant(self, variant_index, material_data):
         var_name = variants[variant_index]
         material_data.name = "hdrihaven/" + base_name + '/' + var_name
 
-        url = "https://hdrihaven.com" + variant_data[variant_index].attrib['href']
+        url = variant_data[variant_index].attrib['href']
```

Comment:
Is this change related?

Reply:
no that is just a mistake sorry about that. this fix just is a temporary fix for cco materials because they had errors on their API JSON page that messed with the JSON parser.